src/components/Album.css

```diff
@@ -0,0 +1,111 @@
+body{
+  background-color: black;
+  font-family: Helvetica, sans-serif;
+}
+/* ----------- Style for header ------------  */
+.header{
+  text-align: center;
+  color: #fff;
+}
+.header h1{
+  font-size: 20px;
+  margin-bottom: 35px;
+}
+/* ----------- Style for all albums ------------  */
+.allAlbums{
+  color: #fff;
+  display: grid;
+  grid-template-columns: 1fr ;
+  padding: 30px;
+}
+.eachAlbum{
+  padding: 25px;
+}
+
+/* ----------- Style for pictures and icons ---------- */
+.cover-container {
+position: relative;
+display: grid;
+place-items: center;
+}
+.cover-image{
+  height: auto;
+  max-width: 600px;
+  width: 100%;
+}
+.cover-container:hover .cover-image {
+  opacity: 0.5;
+  transition: all 0.35s ease-in-out; /* Makes the transition easier */
+
+}
+
+.icon{
+  width: 35px;
+}
+.icon-play{
+  width: 45px;
+}
+
+.icon:hover{
+  width: 40px;
+}
+.icon-play:hover{
+  width: 50px;
+}
+
+.icon-container {
+  position:absolute;
+  display: flex;
+  align-items: center;
+  gap:20px;
+  z-index: 1;
+  opacity: 0 ;
+}
+.cover-container:hover .icon-container {
```

Comment:
I wonder how it would be if you added the "ease-in-out" transformation for the opacity change - maybe it would feel more smoooooth